src/ardupilot/ArduPilotProtocol.h

```diff
@@ -0,0 +1,101 @@
+#include "../network/websocket/WebSocketProtocol.h"
+#include "../network/websocket/WebSocketServer.h"
+#include "../world_interface/data.h"
+
+#include <condition_variable>
+#include <mutex>
+
+namespace net {
+namespace ardupilot {
+
+class ArduPilotProtocol {
+public:
+	ArduPilotProtocol();
+	~ArduPilotProtocol();
+	ArduPilotProtocol(const ArduPilotProtocol& other) = delete;
+	ArduPilotProtocol& operator=(const ArduPilotProtocol& other) = delete;
+
+	// Registers the protocol with the websocket server
+	void initArduPilotServer();
+	// Connection handler for websocket server
+	void clientConnected();
+	// Disconnect handler for websocket server
+	void clientDisconnected();
```

Comment:
Do these methods need to be public? Client code probably doesn't need to be able to call these methods